Dockerfile

```diff
@@ -4,6 +4,9 @@ FROM mambaorg/micromamba:bullseye
 ###################
 ARG PYTHON_VERSION=3.*
 ARG MICRO_TETHYS=false
+ARG DJANGO_VERSION=4.2.*
+ARG DJANGO_CHANNELS_VERSION
+ARG DAPHNE_VERSION
```

Comment:
Can args be declared without a value?

Reply:
Yes, they can be. They actually help with the if statement, if it is not defined then it will not enter the if statement